packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -264,9 +284,11 @@ class ConnectionStateHandler implements IConnectionStateHandler {
         );
     }
 
-    private startJoinOpTimer() {
+    private startJoinOpTimer(writeConnection: boolean) {
         assert(!this.joinOpTimer.hasTimer, 0x234 /* "has joinOpTimer" */);
-        this.joinOpTimer.start();
+        this.joinOpTimer.start(
+            writeConnection ? JoinOpTimeoutMs : JoinSignalTimeoutMs,
```

Comment:
For a write connection does addMember not fire when the join signal arrives, but only when the join op is processed?

Reply:
Ok yeah I found the code. Makes sense to stay tied to quorum for write.  Will be great when the future idea of unifying those datastructures is done!